public/index.html

```diff
@@ -6,23 +6,69 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="description" content="">
     <meta name="author" content="Virtual Video Game Orchestra">
-    <meta name="generator" content="Jekyll v3.8.6">
 
     <title>Virtual Video Game Orchestra</title>
 
     <link rel="icon" href="images/favicons/favicon-32x32.png" sizes="32x32" type="image/png">
+    <link rel="stylesheet" type="text/css" href="npm/bootstrap/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="npm/@fortawesome/fontawesome-free/css/all.min.css">
     <link rel="stylesheet" type="text/css" href="css/theme.css">
+    <link rel="stylesheet" type="text/css" href="css/crt-tv.css">
+
+    <script type="text/javascript" src="npm/jquery/dist/jquery.min.js"></script>
+    <script type="text/javascript" src="npm/bootstrap/dist/js/bootstrap.min.js"></script>
+    <script type="text/javascript" src="npm/@fortawesome/fontawesome-free/js/fontawesome.min.js"></script>
 </head>
 
+<div class="container">
```

Comment:
```suggestion
<div class="container px-0">
```
Kill the padding. We're not using it as part of a grid, so it's not needed. It also looks off at full-width.